src/utils/decorators/MusicUtil.ts

```diff
@@ -1,36 +1,49 @@
 import { createEmbed } from "../createEmbed";
```

Comment:
```suggestion
import { CommandContext } from "../../structures/CommandContext";
```

Reply:
Done in [`7942706`](https://github.com/zhycorp/disc-11/pull/594/commits/7942706ed4bb19192b9e54a2d87852ffefa4d927)